packages/tree-builder/test/applications/download_airdrop_record.test.ts

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import { consolidateRecords } from '../../src/applications/download-airdrop-record/download_airdrop_record';
+
+describe('downloadAirdropRecord', () => {
+	describe('consolidateRecords', () => {
+		it('should group the value of claimedAmount for the same user', () => {
+			const subgraphResponse = [
+				{
+					ID: '1',
+					lskAddress: '0x123',
+					recipient: '0xabc',
+					amount: '10000',
+				},
+				{
+					ID: '2',
+					lskAddress: '0x456',
+					recipient: '0xabc',
+					amount: '20000',
+				},
+				{
+					ID: '3',
+					lskAddress: '0x789',
+					recipient: '0xdef',
+					amount: '30000',
+				},
+			];
+
+			const result = consolidateRecords(subgraphResponse);
+
+			// https://www.wolframalpha.com/input?i=%2810000+*+10+**+18%29+*+%281234567+*+10+**+18%29+%2F+%283000000+*+10+**+18+-+1234567+*+10+**+18%29
```

Comment:
I don't see why this comment is needed here

Reply:
It could be difficult to have math module which comes in handy that supports such precision, so I provider a quick way for reviewers (and others developers) to verify the calculation. If you think it is a not necessary I can remove it.

Reply:
Thank you for pointing that out, I think I accidentally pasted it twice, sorry.